### PR TITLE
Add some changes to make compile for react.

### PR DESCRIPTION
**Pull request by AurelienFT**

### CODE REVIEW

**Comment by AurelienFT** [comment]
Current bug : 
```
Uncaught (in promise) TypeError: base58PublicKey.toBuffer is not a function
    at Function.<anonymous> (WalletClient.ts:223:1)
    at Generator.next (<anonymous>)
    at fulfilled (tslib.es6.js:73:1)
```
Apparently toBuffer doesn't exist on browser. Need to find a workaround

**Comment by AurelienFT** [comment]
Potential fix : https://github.com/indutny/bn.js/issues/147

File: package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
   "dependencies": {
     "@noble/secp256k1": "1.5.5",
     "axios": "^0.26.1",
+    "base58check": "^2.0.0",
     "bn.js": "^5.2.0",
     "bs58check": "^2.1.2",
```
Comment (qdrn) [inline package.json:36]: Is this one still used ? Maybe remove it from `package.json` otherwise.

Comment (qdrn) [reply package.json:36]: cc @AurelienFT

Comment (AurelienFT) [reply package.json:36]: My bad @BatiGencho cleaned it. ty

**Pull request merged**